src/graph/Executor.cpp

```diff
@@ -195,5 +195,130 @@ Status Executor::checkFieldName(std::shared_ptr<const meta::SchemaProviderIf> sc
     return Status::OK();
 }
 
+Status Executor::createSchema(const std::vector<ColumnSpecification*>& specs,
```

Comment:
I think the snippet code's best place is another class such as SchemaHelper under graph module.   If the method is not common used by all executors,  don't put it here.

Reply:
👍👍，excellent